src/Data/Fin/Base.agda

```diff
@@ -307,3 +322,16 @@ fromℕ≤″ = fromℕ<″
 "Warning: fromℕ≤″ was deprecated in v1.2.
 Please use fromℕ<″ instead."
 #-}
+
+-- Version 2.0
+
+raise = _↑ʳ_
+{-# WARNING_ON_USAGE raise
+"Warning: raise was deprecated in v2.0.
+Please use _↑_ʳ instead."
+#-}
+inject+ = flip _↑ˡ_
+{-# WARNING_ON_USAGE inject+
+"Warning: inject+ was deprecated in v2.0.
+Please use _↑ˡ_ instead."
```

Comment:
Can you add "Note that the argument order has been flipped." to the end of this message? Might as well try and save users as much pain as possible :smile: If any of the argument order has swapped on the deprecated proofs then please add a similar message there as well!